### PR TITLE
Snake Growth

### PR DESCRIPTION
**Pull request by riamittal93**
The snake grows by 2. This value seemed more realistic than 5 but we could change it.

### CODE REVIEW

File: snake.py [hunk 1]
```diff
@@ -127,6 +127,7 @@ def main():
 
         if is_eating(snake, food):
             score += score_per_food
+            snake.length += 2
```
Comment (charvakk) [inline snake.py]: We should use the `Snake.growth` attribute here instead of hard-coding the growth to 2.

**Review by charvakk — CHANGES_REQUESTED** [review]

File: snake.py (hunk 1 above)
Comment (charvakk) [reply snake.py]: Also, it would be better if growth was implemented as a function in the `Snake` class and just called here on the snake object.

Comment (riamittal93) [reply snake.py]: but growth literally has just one task to do. Why would that require a whole function?

Comment (charvakk) [reply snake.py]: It's just better organization

**Review by charvakk — APPROVED** [review]

**Pull request merged**